apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js

```diff
@@ -11,15 +11,25 @@ import './help-center.scss';
 
 function HelpCenterComponent() {
 	const isDesktop = useMediaQuery( '(min-width: 480px)' );
-	const { show } = useSelect( ( select ) => {
+	const { show, hasSeenWhatsNewModal, hasFetchedHasSeenWhatsNewModal } = useSelect( ( select ) => {
+		const store = select( 'automattic/help-center' );
 		return {
-			show: select( 'automattic/help-center' ).isHelpCenterShown(),
+			show: store.isHelpCenterShown(),
+			hasSeenWhatsNewModal: store.hasSeenWhatsNewModal(),
+			hasFetchedHasSeenWhatsNewModal: store.hasFetchedHasSeenWhatsNewModal(),
 		};
 	} );
-	const setShowHelpCenter = useDispatch( 'automattic/help-center' )?.setShowHelpCenter;
+	const { setShowHelpCenter, fetchHasSeenWhatsNewModal } = useDispatch( 'automattic/help-center' );
 	const [ selectedArticle, setSelectedArticle ] = useState( null );
 	const [ footerContent, setFooterContent ] = useState( null );
 
+	// On mount check if the Whats New Modal Seen status exists in state (from local storage), otherwise fetch it from the API.
+	useEffect( () => {
+		if ( ! hasFetchedHasSeenWhatsNewModal ) {
+			fetchHasSeenWhatsNewModal();
+		}
+	}, [ hasFetchedHasSeenWhatsNewModal, fetchHasSeenWhatsNewModal ] );
```

Comment:
Like [what we do](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/block-editor-nux.js#L44-L48) for the Welcome Tour